test/gnuradio/test_adi_gnuradio.py

```diff
@@ -0,0 +1,63 @@
+import pytest
+import pyguit
+import time
+import os
+import shutil
+
+class TestADIGnuradio:
+
+    @classmethod
+    def setup_class(self):
+        '''Setup the pyguit object. Will be called once before the start of test'''
+        self.gui = pyguit.gui()
+        self.gui.attach_openbox()
+        # self.gui.run_ssh_agent()
+        time.sleep(10)
+        try:
+            os.makedirs("results")
+        except FileExistsError:
+            # directory already exists
+            shutil.rmtree('results')
+            os.makedirs("results")
+        
+    @classmethod
+    def teardown_class(self):
+        '''Garbage collector. Will be called once after running of tests'''
+        time.sleep(10)
+        self.gui.dettach_openbox()
+        del self.gui
+
+    @pytest.mark.remote
+    def test_open_app_on_remote(self, ip, delay):
+        '''Test if app opens, and checks main window'''
+        self.gui.open_app(
+            host= ip,
+            user="analog",
+            app_name="gnuradio",
+            path="/usr/bin/gnuradio-companion",
+        )
+        time.sleep(15)
+        #find_main_screen
+        for w in self.gui.get_open_windows():
+            if w:
+                print(self.gui.get_window_title(w))      
+        # find_window_title
+        try:  
+            self.gui.find_window("untitled - GNU Radio Companion")
+        except:
+            print("Application window not found")
+        else:
+            self.gui.find_window("*untitled - GNU Radio Companion")
```

Comment:
Else will run if try clause does not raise an error. Wouldn't the else block raise an exception if the window title is "untitled - GNU Radio Companion" without the asterisk? Isn't it enough to look for "- GNU Radio Companion" for the main window?

Reply:
@cseptimo Just refactor the try:except / else clause and this good to me

Reply:
Will check on this.